docker-compose.yaml

```diff
@@ -106,40 +99,31 @@ services:
       - "15432:5432"
     volumes:
       - ./init/postgres:/docker-entrypoint-initdb.d/
-    deploy:
-      resources:
-        limits:
-          cpus: "1"
-          memory: 500M
 
   mysql:
     image: mysql:8.0
     container_name: playground-mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=mysql
+      - MYSQL_USER=mysql
+      - MYSQL_PASSWORD=mysql
+      - MYSQL_DATABASE=db
     ports:
       - "13306:3306"
     volumes:
       - ./init/mysql:/docker-entrypoint-initdb.d/
-    environment:
-      MYSQL_ROOT_PASSWORD: mysql
-      MYSQL_USER: mysql
-      MYSQL_PASSWORD: mysql
-      MYSQL_DATABASE: db
     command:
       --default-authentication-plugin=mysql_native_password
       --character-set-server=utf8mb4
       --collation-server=utf8mb4_general_ci
       --explicit_defaults_for_timestamp=true
       --lower_case_table_names=1
-    deploy:
-      resources:
-        limits:
-          cpus: "1"
-          memory: 500M
     healthcheck:
-      test: ["CMD-SHELL", "mysqladmin ping -h localhost -p ${MYSQL_ROOT_PASSWORD}"]
+      test: ["CMD-SHELL", "mysqladmin ping -h localhost -pmysql"]
```

Comment:
I think better use ${MYSQL_ROOT_PASSWORD}

Reply:
MYSQL_ROOT_PASSWORD environment variable has been set into mysql  container, but always pop this warning:
<img width="827" alt="image" src="https://github.com/user-attachments/assets/e2b5b076-ab53-4b18-b8b3-ede0a873fbda">
<img width="836" alt="image" src="https://github.com/user-attachments/assets/9fb523e4-13db-4899-81e4-4f442bde1a65">
So I get this way to fix:
<img width="938" alt="image" src="https://github.com/user-attachments/assets/1c007f44-e264-420c-b4e4-51da63094499">